ui/helpers/constants/design-system.ts

```diff
@@ -54,6 +54,7 @@ export enum Color {
 export enum BackgroundColor {
   backgroundDefault = 'background-default',
   backgroundAlternative = 'background-alternative',
+  backgroundAlternativeSoft = 'background-alternative-soft',
```

Comment:
We just released a version of the design-tokens `@metmask/design-tokens@v4.2.0` with updated background colors `background-muted`, `background-muted-hover` and `background-muted-active`. Should this be 

```suggestion
  backgroundMuted = 'background-muted',
  backgroundMutedHover = 'background-muted-hover',
  backgroundMutedActive = 'background-muted-active',
```

You will also need to update the CSS I believe. I'll create a PR